cli.js

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const path = require('path');
+
+// Get the project directory
+const projectDir = process.argv[2] || 'new-nil-project';
+const projectPath = path.join(process.cwd(), projectDir);
+
+console.log(`Creating a new project in ${projectDir}...`);
+
+// Create the directory for the new project
+fs.mkdirSync(projectPath, { recursive: true });
+
+function createPackageJson() {
+    const packageJsonContent = {
+        name: path.basename(projectDir),
+        version: "1.0.0",
+        description: "A new project on the =nil; platform",
+        main: "index.js",
+        scripts: {
+            tests: "npx hardhat test --network nil",
+            lint: "biome check ignition/modules test",
+            "lint:fix": "biome check --write ignition/modules test"
+        },
+        dependencies: {},
+        devDependencies: {
+            "@nilfoundation/hardhat-plugin": "^0.15.0",
```

Comment:
I don't like that we have a new entry where we need to maintain the versions. Usually i don't like "latest" but maybe it's worth here as it's just example project?

Reply:
Sasha is going to make the same pull request in nil repo.